canonical-web-and-design/maas-ui
# Add settings sections

Done:
- Add the nav items and base components for each section of the settings. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1332.

QA:
- Load the app.
- You should see the nav items seen here: https://app.zeplin.io/project/5ce5423a25863c1e0d7abb27/screen/5d1cbd18160fa1aff58546d0
- Clicking on them should show the title of the section.
